docs/Docker.md

```diff
@@ -45,5 +43,17 @@ docker run algorand/conduit init --importer algod --processors filter_processor
 With `conduit.yml` in your current working directory,
 launch the container:
 ```
-docker run -it -v $(pwd)/conduit.yml:/conduit/data/conduit.yml algorand/conduit
+docker run -it -v $(pwd)/conduit.yml:/data/conduit.yml algorand/conduit
 ```
+
+# Mounting the Data Directory
```

Comment:
Maybe switch the this section `# Mounting the Data Directory` with the previous `## Run with conduit.yml`? That's because the previous command is mounting and the later section explains mounting.

Reply:
I was thinking that Mounting the data directory is more involved when it comes to permissions compared to only mounting `conduit.yml`. Maybe both sections should be combined somehow.

Reply:
You can just refer to the next section by saying something like
```
...in your current working directory, launch the container. Notice that we're mounting the `conduit.yml` in our current working directory to `data/conduit.yml` in the container (see the next section):
```

Reply:
Now that I think more about it, isn't it preferred to mount the directories as opposed to the config file?